Python/thread.c

```diff
@@ -114,12 +114,9 @@ PyThread_set_stacksize(size_t size)
 #endif
 }
 
-#ifndef Py_HAVE_NATIVE_TLS
```

Comment:
I'd like to change to directive chain and remove `Py_HAVE_NATIVE_TLS`. In fact, native TLS is required.